### ISSUE TITLE
Syntax specific color scheme

### ISSUE THREAD

**Issue by Neospiro** [facelessuser/ExportHtml#37]
I found great to use different color scheme for differents syntaxes 

Can you overwrite the user color scheme with the syntax specific one when exists ? ty

**Comment by facelessuser** [comment]
The documentation covers your question here: http://facelessuser.github.io/ExportHtml/usage/#exporting-html.  Take a look there and if you have questions about the documentation, post them here.

**Comment by Neospiro** [comment]
I've already saw settings and setted a "white background" config to send in mails. If I don't define colorscheme, your plugin the one defined in "preference.sublime-settings".
I choose this package because it exports code in HTML as we see it in Sublime, using the color scheme and syntax definitions.
Except you forgot the "syntax specific" settings.

Exemple : in `JSON.sublime-setting` I have : 
```
{
    "color_scheme": "Packages/User/SublimeLinter/Aurora (SL).tmTheme",
}
```
And your plugin don't use it, so json export with my "plain text" color scheme, defined in default "Preferences.sublime-setting".


Sorry for my bad english
Love from France

**Comment by facelessuser** [comment]
So let me rephrase to make sure I understand:

You aren't trying to construct a command to load a specific syntax file, but you are having problems with ExportHtml picking up syntax specific color schemes when auto detecting the the current color scheme.

Is that right?

**Comment by Neospiro** [comment]
C'est parfait ! This is perfect

Le mer. 3 févr. 2016 à 17:09, Isaac Muse <notifications@github.com> a
écrit :

**Comment by facelessuser** [comment]
No problem.  This is actually easy to fix.  I know exactly what needs to be done.  The original code was written before I knew about how to get syntax specific color schemes.

**Comment by Neospiro** [comment]
Here ?
https://github.com/facelessuser/ExportHtml/blob/master/ExportHtml.py#L418
:smile:

**Comment by facelessuser** [comment]
The fix would actually be here https://github.com/facelessuser/ExportHtml/blob/master/ExportHtml.py#L485 and here https://github.com/facelessuser/ExportHtml/blob/master/ExportBbcode.py#L172

Instead of getting the color scheme from the global settings, you get it from the view which contains a copy of the settings consisting of global settings + syntax settings + any settings that have been overridden or added by a plugin to that specific view.

Instead of `settings.get('color_scheme')` you would call `self.view.settings().get('color_scheme')`  or something like that.  I've only briefly looked at the issue, but I think that should do it.
